docs/data/sdks/android-kotlin/index.md

```diff
@@ -580,7 +580,7 @@ Follow these steps to use Android Ad ID.
 
 !!!warning "Google Ad ID and Tracking Warning"
 
-    As of April 1, 2022, Google allows users to opt out of Ad ID tracking. Ad ID may return null or error. You can use am alternative ID called [App Set ID](#app-set-id), which is unique to every app install on a device. [Learn more](https://support.google.com/googleplay/android-developer/answer/6048248?hl=en)
+    As of April 1, 2022, Google allows users to opt out of Ad ID tracking. Ad ID may return null or error. You can use am alternative ID called [App Set ID](#app-set-id), which is unique to every app install on a device. [Learn more](https://support.google.com/googleplay/android-developer/answer/6048248?hl=en).
```

Comment:
nit: could be worth it to convert this to an include so we can update it in a single place